server/app.js

```diff
@@ -7,6 +7,7 @@ const helmet = require('helmet');
 const origin = require('./shared/origin');
 const passport = require('passport');
 const path = require('path');
+require('express-async-errors');
```

Comment:
@vladimyr On line 14. there is a require that is assigned to an unused variable. Is there a reason why we would want to do this in different ways at different times or should we consolidate it?

Reply:
I'm all for removing it and just requiring stuff at appropriate place without that eslint pragma above.

Reply:
> @lukasaric Please resolve reported conflicts 😉

@vladimyr unused auth variable problem with unnecessary eslint pragmas fixed :point_down: .
Question, should we refactor auth/passport setup like in boutique, or should we leaving it as it is?

Reply:
If that means new pr go for it 😁

Reply:
:+1: